### PR TITLE
Add git internals to the outline

### LINKED ISSUE #1: Talk about internals
Not sure if we'll have time, but it would be great to talk about how git works under the hood.

@muan What do you think, are they up for it?

### PR DESCRIPTION
**Pull request by johndbritton**
This resolves #1, let's cover some internal topics.

@muan How's this?

:palm_tree:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,8 @@
 * stash
 * rebase
 * reflog
+
+# Git Internals (if we have time)
+* Four objects: commit, tree, tag, blob
+* hashes
+* zlib compression
```
Comment (muan) [inline README.md]: Looks legit. :cat2:

File: README.md [hunk 2]
```diff
@@ -17,3 +17,8 @@
 * stash
 * rebase
 * reflog
+
+# Git Internals (if we have time)
+* Four objects: commit, tree, tag, blob
+* hashes
```
Comment (johndbritton) [inline README.md]: specifically SHA1 and how all objects are fingerprinted.

**Comment by muan** [comment]
:cactus: :computer: :shipit:

**Comment by yanyiyi** [comment]
:+1:

**Pull request merged**